pack.yaml

```diff
@@ -3,7 +3,7 @@ ref: vsphere
 name: vsphere
 description: VMware vSphere
 stackstorm_version: ">=2.9.0"
-version: 0.8.12
+version: 0.8.13
```

Comment:
Because it's not a bugfix but new functionality added, it makes sense to bump a minor pack version instead of patch version, eg `0.9.0` to be compliant with https://semver.org/